vendorconsent/tcf2/metadata.go

```diff
@@ -118,29 +130,38 @@ func (c ConsentMetadata) ConsentLanguage() string {
 	return string([]byte{leftChar + 65, rightChar + 65}) // Unicode A-Z is 65-90
 }
 
+// VendorLegitInterestMaxID returns the vendor legitimate interest max id
+func (c ConsentMetadata) VendorLegitInterestMaxID() uint16 {
```

Comment:
Just added  this function for completeness?

Reply:
Hi, no this function already exists (I rebase with master) just add the missing documentation on it

https://github.com/prebid/go-gdpr/blob/master/vendorconsent/tcf2/metadata.go#L122